tests/functional/UseCase/Table/Import/FromTable/IncrementalImportTableFromTableTest.php

```diff
@@ -0,0 +1,192 @@
+                 ] as $i
+        ) {
+            $quotedValues = [];
+            foreach ($i as $item) {
+                $quotedValues[] = BigqueryQuote::quote($item);
+            }
+            $bqClient->runQuery($bqClient->query(sprintf(
+                'INSERT INTO %s.%s VALUES (%s)',
+                BigqueryQuote::quoteSingleIdentifier($bucketDatabaseName),
+                BigqueryQuote::quoteSingleIdentifier($destinationTableName),
+                implode(',', $quotedValues)
+            )));
+        }
+
+        $cmd = new TableImportFromTableCommand();
+        $path = new RepeatedField(GPBType::STRING);
+        $path[] = $bucketDatabaseName;
+        $columnMappings = new RepeatedField(
+            GPBType::MESSAGE,
+            TableImportFromTableCommand\SourceTableMapping\ColumnMapping::class
+        );
+        $columnMappings[] = (new TableImportFromTableCommand\SourceTableMapping\ColumnMapping())
+            ->setSourceColumnName('col1')
+            ->setDestinationColumnName('col1');
+        $columnMappings[] = (new TableImportFromTableCommand\SourceTableMapping\ColumnMapping())
+            ->setSourceColumnName('col2')
+            ->setDestinationColumnName('col4');
+        $cmd->setSource(
+            (new TableImportFromTableCommand\SourceTableMapping())
+                ->setPath($path)
+                ->setTableName($sourceTableName)
+                ->setColumnMappings($columnMappings)
+        );
+        $cmd->setDestination(
+            (new Table())
+                ->setPath($path)
+                ->setTableName($destinationTableName)
+        );
+        $dedupCols = new RepeatedField(GPBType::STRING);
+        $dedupCols[] = 'col1';
+        $cmd->setImportOptions(
+            (new ImportOptions())
+                ->setImportType(ImportOptions\ImportType::INCREMENTAL)
+                ->setDedupType(ImportOptions\DedupType::UPDATE_DUPLICATES)
+                ->setConvertEmptyValuesToNullOnColumns(new RepeatedField(GPBType::STRING))
+                ->setNumberOfIgnoredLines(0)
+                ->setDedupColumnsNames($dedupCols)
+                ->setTimestampColumn('_timestamp')
+        );
+
+        $handler = new ImportTableFromTableHandler($this->clientManager);
+
+        $handler(
+            $this->projectCredentials,
+            $cmd,
+            []
+        );
+        $ref = new BigqueryTableReflection($bqClient, $bucketDatabaseName, $destinationTableName);
+        // 1 row unique from source, 3 rows deduped from source and destination
+        $this->assertSame(4, $ref->getRowsCount());
+        $this->assertTimestamp($bqClient, $bucketDatabaseName, $destinationTableName);
+        $data = $this->fetchTable(
+            $bqClient,
+            $bucketDatabaseName,
+            $destinationTableName,
+            ['col1', 'col4']
+        );
+        $this->assertEqualsCanonicalizing([
+            [
+                'col1' => '1',
```

Comment:
Ono sa netestuju tie nazvy stlpcov nemalo by sa to tiez testovat ?

Reply:
Jak to myslíš?

Reply:
No ze ked som tam teraz dal `col1` tak ten test presiel takze sa testuju len hodnoty.

Reply:
Jo ja som to nepochopil prave jak si to myslel. Hm tak je divne ze to failen na colx ale ked je tam 0 tak nie tak to je asi takto cajk a testuju sa aj tie nazvy

Reply:
![image](https://user-images.githubusercontent.com/180148/204518037-f80e7b19-2470-4e3a-a563-f1ea11ab3d9c.png)

Reply:
:D 
![image](https://user-images.githubusercontent.com/180148/204518094-0e772ded-cc75-418d-aa1b-eb821185eb0a.png)

Reply:
Rak nebude tam staci assertSame ?

Reply:
Nebude, protože to pořadí je náhodné při tom importu